[EDU-3677] feat: metrics data transferred (#736)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/load-balancer.mdx

With **Load Balancer** active, you may create up to 10 origin addresses for each origin.

---

## Billing

**Load Balancer** has specific metrics that define how the product is billed.

<Button href="https://www.azion.com/en/pricing/#LoadBalancer" text="see pricing"></Button>

### Data Transferred

> Total volume of data transferred by Azion Edge Computing Platform with the Load Balancer module enabled, whether it's delivered through cache or fetched from an origin, calculated in GB.

Reviewer: ```suggestion
The total volume of data transferred by **Azion Edge Computing Platform** with the Load Balancer module enabled, whether it's delivered through cache or fetched from an origin, calculated in GB.
```

> If an edge application transfers data and it has the Load Balance module enabled in it, charges will apply for the pricing of the module.

Reviewer: ```suggestion
If an edge application transfers data and has the **Load Balance** module enabled, charges will apply to the pricing of the module.
```

Reviewer: Balance or Balancer?
